frontend/src/fetchers/activityReports.js

```diff
@@ -39,3 +39,9 @@ export const getCollaborators = async (region) => {
   const collaborators = await get(url);
   return collaborators.json();
 };
+
+export const reviewReport = async (reportId, data) => {
+  const url = join(activityReportUrl, reportId.toString(10), 'review');
```

Comment:
Would doing this improve readability?
```javascript
const DECIMAL_BASE = 10
....
reportId.toString(DECIMAL_BASE)
```

I only mention this b/c of magic numbers, and I had to look up what the extra param in `toString` was. This is also due to me not being familiar with JS that much, so I'll defer to you JS vets if it makes sense to change :)

Reply:
> Would doing this improve readability?

:+1: Yes it would